modin/core/dataframe/base/partitioning/axis_partition.py

```diff
@@ -16,8 +16,10 @@
 from abc import ABC, abstractmethod
 from typing import Any, Callable, Iterable, Optional, Tuple, Union
 
+from modin.logging import ClassLogger
 
-class BaseDataframeAxisPartition(ABC):  # pragma: no cover
+
+class BaseDataframeAxisPartition(ABC, ClassLogger):  # pragma: no cover
```

Comment:
Since we use the methods of this class inside remote functions, how does this addition of ClassLogger here affect logging for remote functions?

Reply:
As I understand BaseDataframeAxisPartition's methods  would call the remote functions, I could not see the other way around where methods of BaseDataframeAxisPartition are called in remote functions, I could see data of the partitions are send to remote functions and operations are performed on them, Is there anyplace where partitions would be directly passed to remote functions?

When running remote functions from the methods of BaseDataframeAxisPartition the class logger would work normally as it would log BaseDataframeAxisPartition methods and leave out logging of remote functions.

Reply:
Got it Thanks

When we enable the logger with `LogMode.enable() ` in modin code, the logger gets enabled only on the main process the logger remains disabled in workers so the remote functions execute the function without logging so info about the function wouldn't be logged from remote function.